src/lib/db/clients/mysql.js

```diff
@@ -292,6 +295,41 @@ export async function listDatabases(conn, filter) {
     .map((row) => row.Database);
 }
 
+export async function listCharsets(conn) {
+  const sql = 'show character set';
+
+  const { data } = await driverExecuteQuery(conn, { query: sql });
+
+  return data
+      .map((row) => row.Charset);
+}
+
+export async function listCollations(conn, charset) {
+  const sql = 'show collation where charset = ?';
+
+  const params = [
+    charset,
+  ];
+
+  const { data } = await driverExecuteQuery(conn, { query: sql, params });
+
+  return data
+      .map((row) => row.Collation);
+}
+
+export async function createDatabase(conn, databaseName, charset, collation) {
+  const sql = 'create database ' + databaseName + ' character set ' + charset + ' collate ' + collation;
```

Comment:
you should able to do it with backticks, then use `wrapIdentifier`, see for example truncateAllTables

Reply:
done